en-US/firefox-ios.xliff

```diff
@@ -385,6 +385,23 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Client/DownloadsPanel.strings" source-language="en" datatype="plaintext" target-language="en">
```

Comment:
@justindarc These two strings should not be in a separate `DownloadsPanel` table. Instead no table should be specified and the individual string IDs should be prefixed with something like `Downloads.Panel.*` or so.